README.rst

```diff
@@ -124,6 +133,13 @@ You can update ``php-cs-fixer`` through this command:
 
     $ brew upgrade php-cs-fixer
 
+Locally (PHIVE)
+~~~~~~~~~~~~~~~~~~~
```

Comment:
No so fast, this one is also too long ;)

Reply:
oh no, I really really _really_ thought I had this covered, f**k me, this is a lesson learned on my "I wished wasn't needed "-list :(
aaarrggh I'm sorry man, you called me out on it before and still I messed it up, f**k me :(

Reply:
no worry. instead of fixing that, let us run rst / md linters on CI :D